src/js/services/conversation.js

```diff
@@ -199,27 +174,23 @@ export function uploadImage(file) {
 
         if (!isFileTypeSupported(file.type)) {
             dispatch(showErrorNotification(getState().ui.text.invalidFileError));
-            return Promise.reject('Invalid file type');
```

Comment:
Yes, it should definitely return.
I removed the rejection for the same reason we have a catchall in `sendChain`. We're handling the `exception` here and there's no need to bubble it up to the UI